music21/analysis/neoRiemannian.py

```diff
@@ -681,6 +682,18 @@ def testNeoRiemannianCombinations(self):
         c8_T = LRP_combinations(c8, 'LP')
         self.assertEqual(str(c8_T), '<music21.chord.Chord B-3 E-4 G4>')
 
+        ds_min = chord.Chord('B- D# F#')
+        self.assertEqual(ds_min, LRP_combinations(ds_min, "LP", raiseException=False))
```

Comment:
FYI music21 style is single quotes. also, super minor point, but I would be more specific and `assertIs` to demonstrate that we want the exact same object back.